js/coinsbit.js

```diff
@@ -0,0 +1,742 @@
+                //                     type: "limit",
+                //                     price: "46752.63032535",
+                //                     timestamp: 1640031161.544,
+                //                     side: "buy",
+                //                     dealFee: "0",
+                //                     takerFee: "0",
+                //                     makerFee: "0",
+                //                     dealStock: "0",
+                //                     dealMoney: "0"
+                //                 },
+                //                 ...
+                //             ]
+                //         },
+                //         code: 200
+                //     }
+                const resultBids = this.safeValue (responses[0], 'result', {});
+                const resultAsks = this.safeValue (responses[1], 'result', {});
+                const bids = this.safeValue (resultBids, 'orders', []);
+                const asks = this.safeValue (resultAsks, 'orders', []);
+                const response = { 'bids': bids, 'asks': asks };
+                const priceKey = 'price';
+                const amountKey = 'amount';
+                const timestamp = this.milliseconds ();
+                orderbook = this.parseOrderBook (response, symbol, timestamp, 'bids', 'asks', priceKey, amountKey);
+            }
+        }
+        // orderbook['nonce']  <<< TODO >>> -- atm I dont know how to handle that, as the response doesn't have right that property
+        return orderbook;
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        const request = this.handleMarketTypeAndParams ('fetchOHLCV', undefined, params)[1];
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        request['market'] = market['id'];
+        const duration = this.parseTimeframe (timeframe);
+        const maxAllowedDiapason = this.options['maxDiapasonsForTimeframes'][timeframe];
+        let maxRequestedMilliSeconds = null;
+        if (limit === undefined) {
+            maxRequestedMilliSeconds = maxAllowedDiapason * 1000;
+        } else {
+            maxRequestedMilliSeconds = Math.min (this.sum (limit, 1) * duration, maxAllowedDiapason) * 1000;
+        }
+        let response = null;
+        if (market['spot']) {
+            request['interval'] = this.timeframes[timeframe];
+            const now = this.milliseconds ();
+            let startTime = since;
+            let endTime = now + 1000; // At first, Let's add one second
+            if (startTime === undefined) {
+                startTime = endTime - maxRequestedMilliSeconds;
+            } else {
+                endTime = this.sum (startTime, maxRequestedMilliSeconds);
+            }
+            startTime = startTime / 1000;
+            endTime = endTime / 1000;
+            request['start'] = Math.ceil (startTime);
```

Comment:
See how a shorter version with `parseInt` is done in the other exchanges in all such cases.

Reply:
I decided to ceil it up (instead of rounding down, as parseInt does), however, I think in this case it does't much matter, so yes, I'll use `parseInt`